src/esp/bindings_js/bindings.html

```diff
@@ -86,7 +85,7 @@ <h1>Habitat</h1>
       const simenv = new SimEnv(config, episode, 0);
       const agent = simenv.addAgent(agentConfig);
       const task = new NavigateTask(simenv, {
-        canvas: document.getElementById('canvas2d'),
+        canvas: document.getElementById('canvas'),
```

Comment:
I think we can delete canvas since its no longer needed.

Reply:
The canvas element is still needed for "mousedown" event.